### DISCUSSION TITLE
Bienvenue to npds_dune discussions!

### DISCUSSION THREAD

**Discussion by jipexu** [npds/npds_dune#513] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Jireck-npds** [comment]
Hello JPB,

Veux tu qu'on regarde ensemble certains points qu'on listerai ici ? 

Merci de ton retour
